src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +70,11 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getAge() >= fromAge
+                        && (person.getSex() == Person.Sex.MAN ? person.getAge() <= maleToAge
+                        : person.getAge() <= femaleToAge))
```

Comment:
it's hard to read
consider refactoring like
```suggestion
                .filter(person -> person -> person.getSex() == Person.Sex.MAN
                ...
```